.github/workflows/publish-gh-pages.yml

```diff
@@ -0,0 +1,33 @@
+name: publish-gh-pages
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  build_and_lint:
+    runs-on: ubuntu-latest
```

Comment:
we are using telnyx machines to run github actions, like `telnyx-4xlarge`. Not sure if we should follow this or it is fine to use ubuntu if we don't need access to any of the env vars